src/test/kotlin/uk/gov/dluhc/printapi/messaging/ProcessPrintResponseFileMessageListenerIntegrationTest.kt

```diff
@@ -11,10 +14,13 @@ import uk.gov.dluhc.printapi.testsupport.testdata.entity.buildCertificate
 import uk.gov.dluhc.printapi.testsupport.testdata.model.buildPrintResponses
 import java.util.concurrent.TimeUnit
 
+@TestInstance(Lifecycle.PER_CLASS)
 internal class ProcessPrintResponseFileMessageListenerIntegrationTest : IntegrationTest() {
 
-    @Test
-    fun `should fetch remote print response file and delete it`() {
+    // @Test
+    @ParameterizedTest
+    @MethodSource("uk.gov.dluhc.printapi.testsupport.testdata.ApplicationsApiTestSource#isFromApplicationsApiTestSource")
```

Comment:
I think this is a bit overkill, can we use a CSV source instead? Should be able to test true/false/null this way. Same for the other tests where you've added this